Persistence/Contexts/AppDbContext.cs

```diff
@@ -23,14 +24,15 @@ protected override void OnModelCreating(ModelBuilder builder)
             base.OnModelCreating(builder);
 
             builder.Entity<UserRole>().HasKey(ur => new {ur.UserId, ur.RoleId});
-            
+
             // RELEASE SETUP
             // builder.Entity<Release>().Property(r => r.Id).UseHiLo();
             // builder.Entity<Release>().Property(r => r.IsPublic).HasDefaultValue(true);
 
             // ARTICLE SETUP
             // builder.Entity<Article>().Property(a => a.Id).UseIdentityAlwaysColumn();
             // builder.Entity<Article>().Property(a => a.IsPublic).HasDefaultValue(true);
+            // builder.Entity<Article>().Property(a => a.Date).HasDefaultValue(DateTime.UtcNow);
```

Comment:
Koffer e den her med?

Reply:
### Tenkte vi hadde tidligere satt default verdier til ting i DB, men det var det ikke. Anser disse som lagt på "is", og legger dermed denne sammen med de andre.